README.md

```diff
@@ -4,8 +4,14 @@
 
 <p align='center'>
     Simple extensible ASS subtitle editor for Linux
+<br />
+<br />
+<a href="https://travis-ci.org/rr-/bubblesub">
+    <img src="https://travis-ci.org/rr-/bubblesub.svg?branch=master">
+</a>
```

Comment:
Now the HTML is using wrong indentation.

Reply:
Oh, sorry. Done.